src/components/contact-form/contact-form.jsx

```diff
@@ -0,0 +1,78 @@
+import { useState } from 'react';
+import { useForm } from 'react-hook-form';
+import RightArrow from 'svg/right-arrow/right-arrow';
+import styles from './contact-form.module.scss';
+import EmailInput from './email-input';
+import PaintingsInput from './paintings-input';
+import InputShell from './input-shell';
+
+export default function ContactForm() {
+  const [isToggled, setIsToggled] = useState(false);
+  const {
+    register,
+    handleSubmit,
+    setValue,
+    formState: { errors },
+  } = useForm();
+
+  const formInputs = [
+    { name: 'firstName', label: 'First Name' },
+    { name: 'lastName', label: 'Last Name' },
+    {
+      component: (
+        <EmailInput
+          key={'emailInput'}
+          styles={styles}
```

Comment:
You are passing all of the styles for the entire file into this component rather than passing the styles specific for this component. This isn't super ideal,  but also the email input has its own component, can you move the styles from the input into that specific file?